devices/flash-zynq7000/flashdrv.c

```diff
@@ -73,10 +74,10 @@ static int flashdrv_regionFind(addr_t offs, u32 *id)
 }
 
 
-static inline void flashdrv_serializeTxCmd(u8 *buff, const flash_cmd_t *cmd, addr_t offs)
+static inline void flashdrv_serializeTxCmd(u8 *buff, flash_cmd_t cmd, addr_t offs)
```

Comment:
Avoid passing structs as copy, previous version is better

Reply:
Ok, in this case (sizeof struct is 3) it seems that the compiler manages this situation quite nicely. So I leave it up to you, it can stay as is